x-pack/plugins/security/server/audit/audit_events.test.ts

```diff
@@ -163,13 +164,13 @@ describe('#savedObjectEvent', () => {
     expect(
       savedObjectEvent({
         action: AuditAction.GET,
-        savedObject: { type: 'telemetry', id: 'SAVED_OBJECT_ID' },
+        savedObject: { type: 'telemetry', id: 'SAVED_OBJECT_ID', name: 'telemetry_name' },
```

Comment:
Nit: Was there a reason to add the name to some of these tests where we are not validating the contents of the event? Would it be more beneficial to to add the name to a few tests where we compare to a snapshot like above?